canonicalwebteam/blog/django/views.py

```diff
@@ -47,6 +47,19 @@ def index(request):
                 page=page_param,
                 categories=[category_id],
             )
+            if upcoming:
+                events = api.get_category_by_slug("events")
+                webinars = api.get_category_by_slug("webinars")
+                upcoming, _ = api.get_articles(
```

Comment:
We should name this differently from the function argument, or rename the argument.